governator-jersey/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'java'
+
+dependencies {
+    compile project(':governator-core')
+
+    compile 'javax.servlet:servlet-api:2.5'
```

Comment:
Don't make this a compile dependency. It should be providedRuntime. Also consider upgrading to a more modern version of the servlet api.